Landing buttons only full width on mobile

wackerow opened the pull request:
- Addresses "Desktop buttons shouldn't be full width" card

wackerow committed:
landing buttons to flex-wrap

Fixes

samajammin reviewed (changes requested):
If buttons wrap they should be full width

wackerow committed:
Merge branch 'develop' into landing-buttons

wackerow committed:
Update _theme.scss

wackerow committed:
fix for landing page button widths

media query breaks at larger breakpoint, allowing buttons to collapse into full width column orientation before remainder of views switch to mobile. This works here as in this screen width range, these buttons are within a parent div that is 40% of the screen width, so stacking is appropriate.

wackerow committed:
Merge branch 'develop' into landing-buttons
